HCK06_2019_Janelia/projects/PROJECTS.md

```diff
@@ -2,6 +2,11 @@
 
 The lists below provide an overview of specific projects for which at least one or more attendees have signed up for to contribute to. Projects are collaborative in nature. If you are attending the hackathon and are interested in participating in any of the existing projets then please contact the team listed on the project page. 
 
+* **Python/PyNWB**
+
+   * **Visualization and documentation tools:**

```

Comment:
Can you please move the ``[**NWB:N Jupyter notebook widgets**](projects/JupyterWidgets)`` to the ```###Developer Days``` list

Reply:
Yes -- done!